src/misc/lv_anim.c

```diff
@@ -221,58 +225,22 @@ int32_t lv_anim_path_linear(const lv_anim_t * a)
 
 int32_t lv_anim_path_ease_in(const lv_anim_t * a)
 {
-    /*Calculate the current step*/
-    uint32_t t = lv_map(a->act_time, 0, a->time, 0, LV_BEZIER_VAL_MAX);
-    int32_t step = lv_bezier3(t, 0, 50, 100, LV_BEZIER_VAL_MAX);
-
-    int32_t new_value;
-    new_value = step * (a->end_value - a->start_value);
-    new_value = new_value >> LV_BEZIER_VAL_SHIFT;
-    new_value += a->start_value;
-
-    return new_value;
+    return _path_cubic_bezier(a, 0.42, 0.0, 1.0, 1.0);
```

Comment:
I see that these floats are resolved in compile time, but for sake of clarity would you mind using
```c
return _path_cubic_bezier(a, 430, 0, 1024, 1024);
```

Reply:
Do you prefer using a macro to convert the float values? 

This should be a macro since `LV_BEZIER_VAL_MAX` can be changed theoretically.

```
#define LV_BEZIER_VAL(f) ((int32_t)((f) * LV_BEZIER_VAL_MAX))


int32_t lv_anim_path_ease_in(const lv_anim_t * a)
{
    return lv_anim_path_cubic_bezier(a, LV_BEZIER_VAL(0.42), LV_BEZIER_VAL(0), LV_BEZIER_VAL(1.0), LV_BEZIER_VAL(1));
}

```

Reply:
> This should be a macro since LV_BEZIER_VAL_MAX can be changed theoretically.

I see. I think we can safely assume that these are really evaluated in compile time and they won't make the `float` lib added to the binary.